src/pages/new-bridge/TxHistory/index.tsx

```diff
@@ -30,7 +32,7 @@ const TransactionsList = (props: any) => {
 
   return (
     <TableBox>
-      {pageTransactions.length ? (
+      {pageTransactions.length && walletCurrentAddress ? (
```

Comment:
please use`&& chainId`(export from useRainbowContext) rather than `&& walletCurrentAddress`

the issue that the connection status was displayed incorrectly previously on alpha branch may be related to this.

I will investigate later and using `chainId` is currently the best way from my observation.